sonar-plugin-api/src/main/java/org/sonar/api/batch/bootstrap/internal/ProjectBuilderContext.java

```diff
@@ -27,8 +27,10 @@
  * Context that is passed to {@link org.sonar.api.batch.bootstrap.ProjectBuilder} as parameter.
  * Important - plugins must use this class only for unit test needs.
  *
+ * @deprecated since 6.5
  * @since 3.7
  */
+@Deprecated
 public class ProjectBuilderContext implements ProjectBuilder.Context {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)